### ISSUE TITLE
Changing theme does nothing on Cygwin (followed usage instructions)

### ISSUE THREAD

**Issue by rfgamaral** [zsh-users/antigen#500]
I've followed the [usage](https://github.com/zsh-users/antigen#usage) instructions to install Antigen on Cygwin (Windows). My `.zshrc` is exactly like the one on the README and everything initializes and starts just fine the first time I launch my shell after installing Antigen. So far so good. This is my ZSH version:

> zsh --version
> zsh 5.3 (x86_64-unknown-cygwin)

But when I try to edit the `.zshrc` file to change a simple thing like the theme, say change `antigen theme robbyrussell` to `antigen theme revised`, the theme doesn't change. If I delete the `~/.antigen` folder and restart my shell, the new theme will be loaded. If I try to change it again to something else, it won't load.

In other words, it only changes if I delete the `~/.antigen` folder. What could possibly be the issue?

**Comment by desyncr** [comment]
It's because Antigen caches its bundles and themes in `$ANTIGEN_CACHE` on the first start up.

You have to either:

- `antigen reset`

- `ANTIGEN_CHECK_FILES=(/path/to/.zshrc)`

Take a look at the [configuration](https://github.com/zsh-users/antigen#optimizations) section.

Remember you can interactively switch between themes with `antigen theme` command.

**Comment by rfgamaral** [comment]
That's it. Thank you :)
